dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/RequestMetadata.java

```diff
@@ -58,9 +58,7 @@ public DefaultHttp2Headers toHeaders() {
             .set(TripleHeaderEnum.CONTENT_TYPE_KEY.getHeader(), TripleConstant.CONTENT_PROTO)
             .set(HttpHeaderNames.TE, HttpHeaderValues.TRAILERS);
         setIfNotNull(header, TripleHeaderEnum.TIMEOUT.getHeader(), timeout);
-        if (!"1.0.0".equals(version)) {
-            setIfNotNull(header, TripleHeaderEnum.SERVICE_VERSION.getHeader(), version);
-        }
```

Comment:
What's the purpose of this check? Why need to set NULL for `1.0.0`?

Reply:
Why do I get the feeling that this 1.0.0 is a bit confusing with the Dubbo protocol version?
If it's just the service version number, it shouldn't make a difference to 1.0.0.